src/main/java/pl/sda/javagda28/tourmanagingcrud/entity/Event.java

```diff
@@ -16,15 +27,23 @@
     @Id
     @GeneratedValue
     private Long id;
-    @Column(name = "name")
+
     private String name;
-    @Column(name = "date")
+
     private LocalDateTime date;
+    @Column(length = 2000)
+    private String bio;
 
-    @ManyToMany(mappedBy = "events")
+    @ToString.Exclude
+    @ManyToMany(cascade = CascadeType.MERGE, fetch = FetchType.EAGER)
+    @JoinTable(
+            name = "Bands_Events",
+            joinColumns = { @JoinColumn(name = "event_id", referencedColumnName = "id") },
+            inverseJoinColumns = { @JoinColumn(name = "band_id", referencedColumnName = "id") }
```

Comment:
zakładam że to działa :) nazwy tabel raczej takie powinniśmy mieć: "bands_events"